docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -106,6 +106,50 @@ Refer to
 [the advanced section of the installation guide](./10-installing-kargo.md#advanced-installation)
 for more details.
 
+:::note
+Operators must manually ensure Kargo controllers receive read-only access
+to `Secret`s in the designated namespaces. For example, if `kargo-global-creds`
+is designated as a global credentials namespace, the following `RoleBinding`
+should be created within that `Namespace`:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+    name: kargo-controller-read-secrets
+    namespace: kargo-global-creds
+subjects:
+    - kind: ServiceAccount
+      name: kargo-controller
+      namespace: kargo
+roleRef:
+    kind: Role
```

Comment:
I tested this RoleBinding and for me it only worked if I reference to a 'ClusterRole' because there is no 'Role' kargo-controller-read-secrets but only a 'ClusterRole' kargo-controller-read-secrets

Reply:
> while ClusterRole grants permissions across all namespaces

Just fyi, that's not quite what it does here. I think this seems to be a lesser-known use of ClusterRoles, but when a namespaced `RoleBinding` references a `ClusterRole`, it gets the permissions of that `ClusterRole` _but only in that namespace_.

This is used to avoid having to redefine common sets of permissions in every namespace where they happen to be needed.